diff --git a/chuck/include/chuck_oo.h b/chuck/include/chuck_oo.h
index 0811988..08f777f 100644
--- a/chuck/include/chuck_oo.h
+++ b/chuck/include/chuck_oo.h
@@ -80,11 +80,11 @@ public:
 
 public:
     // add reference (ge: april 2013: made these virtual)
-    virtual void add_ref();
+    virtual void add_ref() {};
     // release reference
-    virtual void release();
+    virtual void release() {};
     // lock
-    virtual void lock();
+    virtual void lock() {};
     
     // NOTE: be careful when overriding these, should always
     // explicitly call up to ChucK_VM_Object (ge: 2013)
@@ -105,7 +105,7 @@ public:
     std::vector<Chuck_VM_Object *> * m_v_ref;
 
 private:
-    void init_ref();
+    void init_ref() {};
 };
 
 
@@ -158,8 +158,8 @@ public:
 struct Chuck_Object : Chuck_VM_Object
 {
 public:
-    Chuck_Object();
-    virtual ~Chuck_Object();
+    Chuck_Object() {};
+    virtual ~Chuck_Object() {};
 
 public:
     // virtual table
